tests/test_miniansegmentationextractor.py

```diff
@@ -0,0 +1,114 @@
+import shutil
+import tempfile
+from pathlib import Path
+
+import numpy as np
+import zarr
+from hdmf.testing import TestCase
+from numpy.testing import assert_array_equal
+
+from roiextractors import MinianSegmentationExtractor
+from tests.setup_paths import OPHYS_DATA_PATH
+
+
+class TestMinianSegmentationExtractor(TestCase):
+    @classmethod
+    def setUpClass(cls):
+        folder_path = str(OPHYS_DATA_PATH / "segmentation_datasets" / "minian")
+
+        cls.folder_path = Path(folder_path)
+        extractor = MinianSegmentationExtractor(folder_path=cls.folder_path)
+        cls.extractor = extractor
+
+        cls.test_dir = Path(tempfile.mkdtemp())
+
+        # denoised traces
+        dataset = zarr.open(folder_path + "/C.zarr")
+        cls.denoised_traces = np.transpose(dataset["C"])
+        cls.num_frames = len(dataset["frame"][:])
+        # deconvolved traces
+        dataset = zarr.open(folder_path + "/S.zarr")
+        cls.deconvolved_traces = np.transpose(dataset["S"])
+        # baseline traces
+        dataset = zarr.open(folder_path + "/b0.zarr")
+        cls.baseline_traces = np.transpose(dataset["b0"])
+        # neuropil trace
+        dataset = zarr.open(folder_path + "/f.zarr")
+        cls.neuropil_trace = np.expand_dims(dataset["f"], axis=1)
+
+        # ROIs masks
+        dataset = zarr.open(folder_path + "/A.zarr")
+        cls.image_masks = np.transpose(dataset["A"], (1, 2, 0))
+        cls.image_size = (dataset["height"].shape[0], dataset["width"].shape[0])
```

Comment:
I think in this and some tests below you compare that the internal representation of the extractor is equal to the API.

I think we should compare the API classes to fixed values.